src/DeviceInterface/AppleDeviceInterface.js

```diff
@@ -284,6 +288,39 @@ class AppleDeviceInterface extends InterfacePrototype {
         })
     }
 
+    async addDeviceListeners () {
+        if (this.settings.featureToggles.third_party_credentials_provider) {
+            if (this.globalConfig.hasModernWebkitAPI) {
+                ddgGlobals.ObjectDefineProperty(ddgGlobals.window, 'providerStatusUpdated', {
```

Comment:
have we double-checked that this method doesn't become accessible to page scripts? I can help test this if not. :)

Reply:
Yep, it's not there. 🙂